README.md

```diff
@@ -44,6 +44,7 @@ Once you have it working in the console, write an event listener that will use t
 
 **YOUR NOTES**
 ```
+I thought I'd be able to pass in the toggleColor function to the event listener but it didn't work.
```

Comment:
I talk about the reason for that in this Slack thread, if you're curious: https://flatiron-school.slack.com/archives/C01CUGGE940/p1605717242012800